documentation/asciidoc/topics/xml/infinispan_caches.xml

```diff
@@ -0,0 +1,27 @@
+<infinispan
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:infinispan:config:{schemaversion} https://infinispan.org/schemas/infinispan-config-{schemaversion}.xsd
+                          urn:infinispan:server:{schemaversion} https://infinispan.org/schemas/infinispan-server-{schemaversion}.xsd"
+      xmlns="urn:infinispan:config:{schemaversion}"
+      xmlns:server="urn:infinispan:server:{schemaversion}">
+  <cache-container name="default"
+                   statistics="true">
+    <transport stack="tcp"/>
```

Comment:
`transport` definitions will be ignored as they are overridden by the operator configuration. Without this behaviour it would be possible for the user to break the cluster communication. 

I suggest we remove `transport` from all of our examples to avoid confusion.